todo-list/src/components/TodoForm.js

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from 'react';
+
+const TodoForm = (props) => {
+	const [title, setTitle] = useState('');
+	const [description, setDescription] = useState('');
+	const [status, setStatus] = useState('');
+	const [priority, setPriority] = useState('');
+
+	const handleSubmit = (event) => {
+		event.preventDefault();
+		const task = {
+			title: title,
+			description: description,
+			status: status,
+			priority: priority,
+		};
+
+		console.log(task);
+		props.addNewTask(task);
+	};
+
+	return (
+		<div className="container">
+			<form onSubmit={handleSubmit}>
+				<div className="field">
+					<label class="label">
+						title:
+						<div className="control">
+							<input
+								type="text"
+								value={title}
+								onChange={(e) => setTitle(e.target.value)}
+								className="input"
+							/>
+						</div>
+					</label>
```

Comment:
is there a reason this label is wrapping the rest of the input, while the other labels aren't? if not could think about using the same syntax for the inputs.

Reply:
it was definitely a typo! missed that one out while adding the css classes